pkg/build/generic/docker.go

```diff
@@ -0,0 +1,100 @@
+package generic
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/testground/testground/pkg/api"
+	"github.com/testground/testground/pkg/build/common"
+	"github.com/testground/testground/pkg/docker"
+	"github.com/testground/testground/pkg/rpc"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+const (
+	buildNetworkName = "testground-build"
```

Comment:
Can we make this a package constant? It's used elsewhere.

Reply:
This const is not used in this file.

Reply:
Right,  removed from this file  in the new flattened version.